topi/include/topi/x86/injective.h

```diff
@@ -44,14 +47,29 @@ namespace x86 {
  * \return The updated schedule.
  */
 inline Schedule schedule_injective_from_existing(Schedule sch, const Tensor& out) {
-  auto axis = sch[out]->op.as<ComputeOpNode>()->axis;
-  if (axis.size() == 4) {
-    auto n = axis[0];
-    auto c = axis[1];
-    auto fused = detail::Fuse(sch[out], {n, c});  // for nhwc layout, fuse n and h
-    sch[out].parallel(fused);
+  const auto& axes = sch[out]->op.as<ComputeOpNode>()->axis;
```

Comment:
Is this schedule func used in the common compilation flow? In op strategy we use python version of injective schedule and it should handle parallelism appropriately.